easybuild/easyconfigs/i/imkl/imkl-11.3.3.210-iompi-2016.03b.eb

```diff
@@ -0,0 +1,19 @@
+name = 'imkl'
+version = '11.3.3.210'
+
+homepage = 'http://software.intel.com/en-us/intel-mkl/'
+description = """Intel Math Kernel Library is a library of highly optimized, extensively threaded math routines
+for science, engineering, and financial applications that require maximum performance. Core math functions include
+BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more."""
+
+toolchain = {'name': 'iompi', 'version': '2016.03b'}
+
+sources = ['l_mkl_%(version)s.tgz']
+
+dontcreateinstalldir = 'True'
+
+# license file
+import os
+license_file = os.path.join(os.getenv('HOME'), "licenses", "intel", "license.lic")
```

Comment:
@jhein32 please sync this with existing `imkl` easyconfigs, see https://github.com/hpcugent/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/i/imkl/imkl-11.3.3.210-iimpi-2016b.eb

* include `checksums`
* include `components`
* use `HOME` constant and avoiding use of `import`
* add `postinstallcmds`